acl/src/main/java/org/apache/rocketmq/acl/common/AclUtils.java

```diff
@@ -206,61 +206,31 @@ public static boolean isIPv6Scope(int num) {
     }
 
     public static String expandIP(String netaddress, int part) {
-        boolean compress = false;
-        int compressIndex = -1;
-        String[] strArray = StringUtils.split(netaddress, ":");
-        ArrayList<Integer> indexes = new ArrayList<>();
-        for (int i = 0; i < netaddress.length(); i++) {
-            if (netaddress.charAt(i) == ':') {
-                if (indexes.size() > 0 && i - indexes.get(indexes.size() - 1) == 1) {
-                    compressIndex = i;
-                    compress = true;
-                }
-                indexes.add(i);
+        // expand netaddress
+        int separatorCount = StringUtils.countMatches(netaddress, ":");
+        int padCount = part - separatorCount;
+        if(padCount > 0){
```

Comment:
make sure the checkstyle pass. run `mvn -Prelease-all -DskipTests install -U` to check build